internal/integration/integration.go

```diff
@@ -121,7 +120,7 @@ type Tester struct {
 	wg       sync.WaitGroup // used for shutdown
 }
 
-const commandLine = `start --masterkeyFromEnv`
+const commandLine = `start-from-init --masterkeyFromEnv`
```

Comment:
Without this, the database isn't initialize so tables are missing.

I am not sure how I suppose to run the integration test, but this was the only way I made it work running things from scratch

Reply:
Have you read the above link I posted in the contribution guidelines and integration tests? The setup is done from the command line/ Makefile, so that we don;t have to re-run setup for every package being tested.

Reply:
I see what was the issue here. You are expecting people to `always` use the make file, but I was using my IDE to run the Go test